sdk/monitor/monitor-opentelemetry-exporter/src/utils/common.ts

```diff
@@ -287,3 +287,7 @@ export function serializeAttribute(value: AnyValue): string {
 export function shouldCreateResourceMetric(): boolean {
   return !(process.env[ENV_OPENTELEMETRY_RESOURCE_METRIC_DISABLED]?.toLowerCase() === "true");
 }
+
+export function isSyntheticSource(attributes: Attributes): boolean {
+  return !!attributes[experimentalOpenTelemetryValues.SYNTHETIC_TYPE];
```

Comment:
The current discussion about the "changing" of the imported semantic conventions is interesting and is going to affect this specific code implementation....

As as soon as this "changes" (or goes stable and is no longer in the "experimental" set) this will be broken, and there is no easy answer for this (hopefully it won't change...

But maybe we should have a "fallback" here so we don't have to panic in the future.

```ts
return !!attributes[experimentalOpenTelemetryValues.SYNTHETIC_TYPE || "user_agent.synthetic.type"];
```

This would cover 
- the experimental value changing
- it staying the same but moving to stable (out of experimental)

But it would not cover it changing AND moving out of experimental.

Reply:
`experimentalOpenTelemetryValues` is an internally defined enum, so we're basically already using the static string you show here. This is consistent between us and Python to never rely on pulling experimental attributes directly from semantic conventions.